### PR TITLE
Kubernetes YAML Addition

### PR DESCRIPTION
**Pull request by drewjj**
This pull request includes full Kubernetes support for the CV Manager Webapp, API and two of its main dependency add-ons.
- The port number for the API has been changed due to health check and Ingress related reasons
-  Health check endpoint has been added to the CV Manager API for K8s health check support

### CODE REVIEW

File: api/src/middleware.py [hunk 1]
```diff
@@ -54,6 +54,10 @@ def __call__(self, environ, start_response):
     # Do not bother authorizing a CORS check
     if request.method == "OPTIONS":
       return self.app(environ, start_response)
+
```
Comment (payneBrandon) [inline api/src/middleware.py]: Now that we have several "do not authorize" checks in here, should we abstract that to a new method for readability?

File: documents/kubernetes/cv-manager-postgres.yaml [hunk 2]
```diff
@@ -50,47 +49,49 @@ metadata:
...
             limits:
-              memory: "4Gi"
-              cpu: "1"
-              ephemeral-storage: "4Gi"
+              memory: '4Gi'
+              cpu: '1'
+              ephemeral-storage: '4Gi'
           tty: true
           stdin: true
           env:
```
Comment (payneBrandon) [inline documents/kubernetes/cv-manager-postgres.yaml:93]: Let's add in a comment like with the new cv-manager-api.yaml to state these are what needs to be updated for their environment.

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Looking good! As we talked about offline, let's get a CORS policy in place for the API here, and I think we're good to go

File: documents/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Kubernetes Deployment Scripts
+
+The CV Manager supports being hosted within a Kubernetes cluster which allows for better stability, smoother deployments and performance scaling. The YAML files within this directory provide a starting point to incorporate the CV Manager into your own Helm deployments in any local or cloud based Kubernetes environment.
+
```
Comment (mwodahl) [inline documents/kubernetes/README.md:4]: Is it worth making a note here that the example YAML files are currently set up to use GCP?

File: api/src/middleware.py (hunk 1 above)
Comment (drewjj) [reply api/src/middleware.py]: Addressed

File: documents/kubernetes/cv-manager-postgres.yaml (hunk 2 above)
Comment (drewjj) [reply documents/kubernetes/cv-manager-postgres.yaml:93]: Addressed

File: documents/kubernetes/README.md (hunk 3 above)
Comment (drewjj) [reply documents/kubernetes/README.md:4]: Addressed

**Comment by drewjj** [comment]
Added CORS support based on the webapp domain. Added a new environment variable to do this.

**Review by mwodahl — APPROVED** [review]
Looks good to me!

File: api/src/middleware.py (hunk 1 above)
Comment (payneBrandon) [reply api/src/middleware.py]: Looks slick!

**Review by payneBrandon — APPROVED** [review]
Looks good!

**Pull request merged**